Chapter9/Chapter9/View/RestaurantDetailHeaderView.swift

```diff
@@ -13,6 +13,7 @@ class RestaurantDetailHeaderView: UIView {
     @IBOutlet private weak var headerViewTopConstraint: NSLayoutConstraint!
     @IBOutlet private weak var heartImageView: UIImageView!
     @IBOutlet private weak var headerImageView: UIImageView!
+    @IBOutlet weak var ratingImageView: UIImageView!
```

Comment:
Make this private as well, if you need to set an image set it through a public method